### PR TITLE
Added support for keybindings in wayland

### PR DESCRIPTION
**Pull request by SoumyaRanjanPatnaik**
image

### CODE REVIEW

File: src/keybindings/I3Ipc.vala [hunk 1]
```diff
@@ -42,15 +46,97 @@ namespace Ilia {
...
+          string[] included_paths = get_includes_from_partial(baseConfig);
+          foreach(unowned string path in included_paths) {
+              path_queue.push_tail(path);
+          }
+
+          while(!path_queue.is_empty()) {
+
+            // Replace head of queue with paths obtained from glob matching
+            // Head of queue doesn't change if the string doesn't contain widcards
+            Posix.Glob pathMatcher = Posix.Glob();
```
Comment (kgilmer) [inline src/keybindings/I3Ipc.vala]: wow, nice!

File: src/keybindings/I3Ipc.vala [hunk 2]
```diff
@@ -42,15 +46,97 @@ namespace Ilia {
...
+            FileUtils.get_contents (config_path, out config_partial);
+            configBuilder.append(config_partial);
+
+            // Append paths of configs included from current config partial to queue for bfs
+            string[] include_paths = get_includes_from_partial(config_partial);
+            foreach (unowned string path in include_paths) {
+              path_queue.push_tail(path);
+            }
+
+            visited_paths.add(config_path);
```
Comment (kgilmer) [inline src/keybindings/I3Ipc.vala]: I suggest moving this line up  to 99 to keep the uniqueness check all in one spot

**Review by kgilmer — APPROVED** [review]
Fantastic work Soumya

**Comment by SoumyaRanjanPatnaik** [comment]
Thanks. Merging...

**Pull request merged**